examples/1-screenshots.sh

```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env bash
```

Comment:
move scripts from examples/ to root directory

Reply:
done